### PR TITLE
create edit layout

### PR DESCRIPTION
**Pull request by ghost**
## Description
Create edit book layout
## Reference Requirements
- Product Backlog ID: #PBI-4
- Task ID: #4.1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- resources/views/backend/layouts/main.blade.php
- resources/views/backend/layouts/partials/js.blade.php
- resources/views/backend/books/edit.blade.php
- public/js/main.js
- public/css/main.css
- resources/lang/en/book.php

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/backend/books/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    </ol>
+  </section>
+    <!-- Main content -->
+  <section class="content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="box box-info">
+          <!-- /.box-header -->
+          <div class="box-body pad">
+            <form action="{{ route('books.store') }}" method="POST" enctype="multipart/form-data">
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: why route is store

File: resources/views/backend/books/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+              {{csrf_field()}}
+              <div class="form-group">
+                <label for="name">Name</label>
+                <input name="name" type="text" class="form-control" id="name" placeholder="{{ __('book.enter_name') }}" value="{{ old('name') }}">
+                @if($errors->first('name'))
+                  <span class="text-danger">{{ $errors->first('name') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="name">{{ __('book.category') }}</label>
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: for="category"

File: resources/views/backend/books/edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+              <div class="form-group">
+                <label for="year">{{ __('book.year') }}</label>
+                <input name="year" type="text" class="form-control" id="year" placeholder="{{ __('book.enter_year') }}" value="{{ old('year') }}">
+                @if($errors->first('year'))
+                  <span class="text-danger">{{ $errors->first('year') }}</span>
+                @endif
+              </div>
+              <div class="form-group">
+                <label for="description">{{__('book.description') }}</label></br>
+                <textarea class="ckeditor" id="description" placeholder=""></textarea>
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: add name textarea and add code
@if($errors->first('description'))
                   <span class="text-danger">{{ $errors->first('description') }}</span>
@endif

File: resources/views/backend/books/edit.blade.php [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                <input name="image" type="file" id="image">
+                @if($errors->first('image'))
+                  <span class="text-danger">{{ $errors->first('image') }}</span>
+                @endif
+              </div>
+
+              <div class="box-footer">
+                <button id="btn-add-book" type="submit" class="btn btn-primary">{{ __('book.submit') }}</button>
+                <button type="reset" class="btn btn-danger">{{ __('book.reset') }}</button>
+                <button id="btn-back" type="reset" class="btn btn-warning">{{ __('book.back') }}</button>
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: how to use back button?

File: resources/views/backend/books/edit.blade.php [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+  </section>
+    <!-- Main content -->
+  <section class="content">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="box box-info">
+          <!-- /.box-header -->
+          <div class="box-body pad">
+            <form action="" method="POST" enctype="multipart/form-data">
+              {{csrf_field()}}
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: {{ method_filed('PUT')}}

File: resources/views/backend/books/edit.blade.php [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+              <div class="form-group">
+                <label for="name">Name</label>
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: localization Name

File: resources/views/backend/books/edit.blade.php [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                <input name="image" type="file" id="image">
+                @if($errors->first('image'))
+                  <span class="text-danger">{{ $errors->first('image') }}</span>
+                @endif
+              </div>
+
+              <div class="box-footer">
+                <button id="btn-add-book" type="submit" class="btn btn-primary">{{ __('book.submit') }}</button>
+                <button type="reset" class="btn btn-danger">{{ __('book.reset') }}</button>
+                <a id="btn-back" href="{{ URL::previous() }}" class="btn btn-warning">{{ __('book.back') }}</a>
```
Comment (at-annguyen) [inline resources/views/backend/books/edit.blade.php]: should change class from btn-warning to btn-default

**Comment by at-annguyen** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**